ApplicationLibCode/ProjectDataModel/CMakeLists_files.cmake

```diff
@@ -67,6 +67,7 @@ set(SOURCE_GROUP_HEADER_FILES
     ${CMAKE_CURRENT_LIST_DIR}/RimContextCommandBuilder.h
     ${CMAKE_CURRENT_LIST_DIR}/RimGridCollection.h
     ${CMAKE_CURRENT_LIST_DIR}/RimPlotCurve.h
+    ${CMAKE_CURRENT_LIST_DIR}/RimPlotCurveCurve.h
```

Comment:
Hmmm, does this file exist? Should it be RimPlotCurveAppearance?

Reply:
Fixed.